tests/components/teslemetry/test_sensor.py

```diff
@@ -72,7 +72,7 @@ async def test_sensors_streaming(
                 Signal.AC_CHARGING_ENERGY_IN: 10,
                 Signal.AC_CHARGING_POWER: 2,
                 Signal.CHARGING_CABLE_TYPE: None,
-                Signal.TIME_TO_FULL_CHARGE: 10,
+                Signal.TIME_TO_FULL_CHARGE: 0.166666667,
```

Comment:
is this nicer for end users? It's hard to process in the UI too. Why not use a timestamp entity so the frontend can render "In 5 minutes" or "in 1 hour"?

Look at the print finish sensor for Prusa https://github.com/home-assistant/core/blob/dev/homeassistant/components/prusalink/sensor.py#L182-L185

Reply:
This is a timestamp sensor, it's converted to one using the provided minutes + dt_util.now()

I was told a while ago time countdown entities were not allowed.